libs/feature/search/src/lib/state/effects.spec.ts

```diff
@@ -483,6 +483,21 @@ describe('Effects', () => {
             )
           })
         })
+        describe('when geometry is undefined', () => {
+          beforeEach(() => {
+            effects['filterGeometry$'] = of(undefined) as any
+            effects = TestBed.inject(SearchEffects)
+            actions$ = of(new RequestMoreResults('main'))
+          })
+          it('skips the geometry in the search', async () => {
+            await firstValueFrom(effects.loadResults$)
+            expect(repository.search).toHaveBeenCalledWith(
+              expect.not.objectContaining({
```

Comment:
could you be more specific in your assertion here and make sure that `geom` was null?

Reply:
As commented above, I made sure it's `undefined` in the test.